pages/product/[productAppExtension]/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productAppExtension = Number(router.query?.productAppExtension);
+    const { error, isLoading, list = [] } = useProductList();
+    const { isLoading: isInfoLoading, product } = useProductInfo(productAppExtension, list);
+    const { description, is_visible: isVisible,name, price, type,} = product ?? {};
+    
+    const productType = type?.replace(/^\w/, (c: string) => c.toUpperCase());
+
+    if (isLoading || isInfoLoading) return <Loading />;
+    if (error) return <ErrorMessage error={error} />;
+
+  return (
+        <>
+        <Panel header="Basic Information">
+            <Text bold>Product name</Text>
+            <Text>{name}</Text>
+            <Text bold>Product type</Text>
+            <Text>{productType}</Text>
+            <Text bold>Default price (excluding tax)</Text>
+            <Text>${price}</Text>
+            <Text>{isVisible}</Text>
+        </Panel>
+        <Panel header="Description">
+            <Text bold>Description</Text>
+            <Text>{description}</Text>
+        </Panel>
+    </>
+  );
```

Comment:
the spacing here is still wonky

```suggestion
    return (
        <>
            <Panel header="Basic Information">
                <Text bold>Product name</Text>
                <Text>{name}</Text>
                <Text bold>Product type</Text>
                <Text>{productType}</Text>
                <Text bold>Default price (excluding tax)</Text>
                <Text>${price}</Text>
                <Text>{isVisible}</Text>
            </Panel>
            <Panel header="Description">
                <Text bold>Description</Text>
                <Text>{description}</Text>
            </Panel>
        </>
    );
```